backend/src/routes/metadata.ts

```diff
@@ -0,0 +1,108 @@
+import { GetSecretValueCommand, SecretsManagerClient } from "@aws-sdk/client-secrets-manager";
+import { FastifyPluginAsync } from "fastify";
+import { Wallet, providers } from "ethers";
+import SupportedNetworks from "../../config.json" assert { type: "json" };
+import { getNetworkConfig, printRequest, getSQLdata } from "../utils/common.js";
+import ReturnCode from "../constants/ReturnCode.js";
+import ErrorMessage from "../constants/ErrorMessage.js";
+import { decode } from "../utils/crypto.js";
+import { PAYMASTER_ADDRESS } from "../constants/Pimlico.js";
+
+
+const metadataRoutes: FastifyPluginAsync = async (server) => {
+
+  const prefixSecretId = 'arka_';
+
+  let client: SecretsManagerClient;
+
+  const unsafeMode: boolean = process.env.UNSAFE_MODE == "true" ? true : false;
+
+  if (!unsafeMode) {
+    client = new SecretsManagerClient();
+  }
+
+  server.get('/metadata', async function (request, reply) {
+    try {
+      printRequest(request, server.log);
```

Comment:
Please see the latest version of the code, this function parameters have changed.